.template/addons/github/.github/workflows/test.yml.tt

```diff
@@ -0,0 +1,76 @@
+name: Test
+
+on: push
+
+env:
+  DOCKER_REGISTRY_HOST: ${{ secrets.DOCKER_REGISTRY_HOST }}
+  DOCKER_REGISTRY_USERNAME: ${{ github.repository_owner }}
+  DOCKER_REGISTRY_TOKEN: ${{ secrets.DOCKER_REGISTRY_TOKEN }}
+  COMPOSE_FILE: docker-compose.test.yml
```

Comment:
Do we still need this @longnd 🤔 as I saw you approved this PR already

Reply:
it isn't necessary @andyduong1920 since we haven't used Danger in the workflow yet. There is a warning in the console that the env isn't set so it will fall back to the default value which is blank.